includes/Admin/Settings.php

```diff
@@ -484,13 +511,13 @@ public function get_settings_fields() {
             ),
             'dokan_withdraw'   => [
                 'withdraw_methods'    => [
-                    'name'    => 'withdraw_methods',
-                    'label'   => __( 'Withdraw Methods', 'dokan-lite' ),
-                    'desc'    => __( 'Select suitable Withdraw methods for Vendors', 'dokan-lite' ),
-                    'type'    => 'multicheck',
-                    'default' => [ 'paypal' => 'paypal' ],
-                    'options' => dokan_withdraw_get_methods(),
-                    'tooltip' => __( 'Check to add available payment methods for vendors to withdraw money.', 'dokan-lite' ),
```

Comment:
Why these tooltips are removed?

Reply:
Because, tooltip is not managed in our new design...